docs/toc.json

```diff
@@ -36,5 +36,8 @@
   },
   "integrations": {
     "title": "Integrations"
+  },
+  "examples": {
+    "title": "Examples"
```

Comment:
Let's leave it out of the TOC until we add the first example. A bit confusing otherwise.